### DISCUSSION TITLE
토큰 발급 이후 세션 생성은 개별 작업을 해야 하는 건가요 ?

### DISCUSSION THREAD

**Discussion by seiblog-nandsoft** [gnuboard/g6#581] [Q&A]
질문 게시판은 공식 사이트에 있긴 한데, 개발자 분들에게 여쭤봐야 할 것 같아서요

토큰은 발급 잘되는데

python 에는 세션 발급이 잘 이루어지고 있어, 실제 로그인을 해야 하는데

rest_api 에서 토큰 발급을 통해, 로그인이 된 상태를 세션 혹은 다른 방법으로 직접 구현해야 하는 것인지 여쭤보고 싶습니다 ㅠ

죄송합니다. 이슈가 아니라서...

**Comment by KimTom89** [comment]
@seiblog-nandsoft 
REST API의 **무상태(Stateless)** 라는  특성 때문에 **토큰 기반 인증(JWT)** 을 사용하여 로그인 상태를 유지하고 있습니다.
**토큰이 정상적으로 발급**되고, 이를 사용하여 **데이터를 조회**할 수 있으면
서버에서는 해당 사용자를 로그인한 상태로 간주합니다.

정확히 어떤 것을 구현해야하는지 정확하게 파악되진 않지만, 별도로 로그인 상태를 구현하지는 않아도 될 것 같습니다.

**Comment by KimTom89** [comment]
@NamjuPortfolio 
현재 소셜로그인은 API에서 누락되었으며 템플릿에서만 사용가능합니다.

조만간 외부 API(소셜로그인, 본인인증 등)를 연동하는 기능들을 제공할 예정입니다 :)
